### DISCUSSION TITLE
Wrong result or wrong formula?

### DISCUSSION THREAD

**Discussion by seghier** [ForNeVeR/wpf-math#420] [Q&A] [answered]
Hi
What's wrong with this formula?

`Eq(Mq(x), -Integral(q(x), x, x) + Integral(q(x), x, (x, 0)) + x*(Integral(q(l), l, l) - Integral(q(x), x, (x, 0)))/l)`

![image](https://github.com/ForNeVeR/xaml-math/assets/6026588/4ae852d6-10ca-4c2f-b295-f6704baf18ca)

**Comment by ForNeVeR** [comment]
What do you mean by "wrong result"? The render looks correct to me. Is there something I'm missing?

**Reply by seghier** [reply, marked as answer]
My mistake i didn't see any changes in the result and i though that integral must rendered like this
![image](https://github.com/ForNeVeR/xaml-math/assets/6026588/6e221bd4-d1b3-4cf5-a2ce-69af3a8a0cba)
